### DISCUSSION TITLE
Google Consent Mode disponible ?

### DISCUSSION THREAD

**Discussion by meytoof** [AmauriC/tarteaucitron.js#942] [General]
Bonjour AmauriC,

Un grand merci pour Tarte au Citron qui est vraiment un outil genial à utiliser !

Est ce que tu penses que tac aura un jour une implémentation Google Consent Mode ?
Ce serait vraiment trés intéressant !

**Comment by AmauriC** [comment]
Bonjour  : )

Il y a très peu de chance que ce soit dispo sur tarteaucitron vu que tac fait déjà le travail de blocage/consentement et que c’est toujours un peu flou avec Google sur le fonctionnement du « consent mode ».
